build.gradle

```diff
@@ -325,6 +327,9 @@ dependencies {
     // Required for SV Discovery machine learning
     compile group: 'biz.k11i', name: 'xgboost-predictor', version: '0.3.0'
 
+    // natural sort
+    compile('net.grey-panther:natural-comparator:1.1')
```

Comment:
Let's spend another 15 minutes or so trying to find a way to avoid adding this dependency...

Reply:
Also, why can't we just use sequence dictionary order for the contigs? That seems safer, and we should have a sequence dictionary from the reference in this tool...

Reply:
This is for gene names, not contigs.  We have plenty of GATK code for sorting contigs.

No action.